appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -4344,6 +4344,22 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String VibrateMethods();
 
+  @DefaultMessage("VibratePattern")
```

Comment:
Adding the component properties/methods/event names to OdeMessages is no longer necessary. One of the annotation processors generates an interface AutogeneratedOdeMessages, a superclass of OdeMessages, with all of this information. There are still some vestigial references in OdeMessages though.

Reply:
Done.